packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -226,65 +229,68 @@ class ImagePickerPlugin extends ImagePickerPlatform {
     return null;
   }
 
-  List<html.File>? _getFilesFromInput(html.FileUploadInputElement input) {
+  List<web.File>? _getFilesFromInput(web.HTMLInputElement input) {
     if (_hasOverrides) {
       return _overrides!.getMultipleFilesFromInput(input);
     }
-    return input.files;
+    final web.FileList? fileList = input.files;
+    final List<web.File> files = <web.File>[];
+    for (int i = 0; i < (fileList?.length ?? 0); i++) {
+      files.add(input.files!.item(i)!);
+    }
```

Comment:
I think this can be `.cast()`

Reply:
And/or use collection syntax here!

Reply:
Can't be cast, because `web.FileList` is very very limited. I fixed this with collection syntax + extension method on `web.FileList`, looks fancy!